packages/nocodb-sdk/src/lib/columnHelper/columns/User.ts

```
@@ -0,0 +1,38 @@
+
+export class UserHelper extends AbstractColumnHelper {
+  columnDefaultMeta = {};
+
+  serializeValue(
+    value: any,
+    params: SerializerOrParserFnProps['params']
+  ): string | null {
+    try {
+      value = typeof value === 'string' ? JSON.parse(value) : value;
+    } catch {
+      value = value;
```

Biome (1.9.4)

[error] value is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)